WcaOnRails/app/views/competitions_mailer/notify_users_of_results_presence.html.erb

```diff
@@ -1,16 +1,15 @@
 <p>
-  Your results at <%= link_to @competition.name, competition_url(@competition) %> have just been posted.
+  <%= t 'users.mailer.results_presence_email.intro_html', competition: @competition.name, competition_url: competition_url(@competition) %>
```

Comment:
In general, when possible, we'd rather avoid presenting the translators links to deal with: we can't assume they all understand how html works, and our translation helping tool doesn't check that html is well formed.
However it does check that variables in strings are still present after translation, so I'd rather do something like this for translations with links:
```ruby
<%= t 'users.mailer.results_presence_email.intro_html', competition_link: link_to(@competition.name, competition_url(@competition)) %>
```
And in the key:
```yaml
Your results at %{competition_link} have just been posted.
```
You may have to use `competition_link: raw(link_to(...))`, IIRC variables are escaped when passed to keys.